src/test/java/com/laserfiche/api/client/integration/BaseTest.java

```diff
@@ -8,10 +8,17 @@
 public class BaseTest {
     protected static String spKey;
     protected static AccessKey accessKey;
-
+    protected static String repoId;
+    protected static String username;
+    protected static String password;
+    protected static String baseUrl;
     @BeforeAll
     public static void setUp() {
         spKey = System.getenv("SERVICE_PRINCIPAL_KEY");
+        repoId = System.getenv("REPOSITORY_ID");
```

Comment:
Reminder: we probably also want to add these in the GitHub pipeline in the future.

Reply:
so i addressed this with Paolo in our mobbing session last friday, and he suggested that we should run these on tfs instead of github and that i should create a low priority story for it which i will get to later